Example-Crypto-REXX/RXCRYPT1.rexx

```diff
@@ -0,0 +1,2431 @@
+/* REXX */
+/* START OF SPECIFICATIONS *********************************************/
+/* Beginning of Copyright and License                                  */
+/*                                                                     */
+/* Copyright 2022 IBM Corp.                                            */
+/*                                                                     */
+/* Licensed under the Apache License, Version 2.0 (the "License");     */
+/* you may not use this file except in compliance with the License.    */
+/* You may obtain a copy of the License at                             */
+/*                                                                     */
+/* http://www.apache.org/licenses/LICENSE-2.0                          */
+/*                                                                     */
+/* Unless required by applicable law or agreed to in writing,          */
+/* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,        */
+/* either express or implied.  See the License for the specific        */
+/* language governing permissions and limitations under the License.   */
+/*                                                                     */
+/* End of Copyright and License                                        */
+/***********************************************************************/
+/*                                                                     */
+/*  SCRIPT NAME=RXCRYPT1                                               */
+/*                                                                     */
+/*  DESCRIPTIVE NAME:                                                  */
+/*    Sample REXX code that uses HWIREST API to query Image            */
+/*    Activation Profile information.                                  */
+/*    It also takes advantage of the JSON Parser to retrieve           */
+/*    various pieces of the information.                               */
+/*                                                                     */
+/* OPERATION:                                                          */
+/*                                                                     */
+/*  CODE FLOW in this sample:                                          */
+/*    Call List CPCs to retrieve uri and target name for the specific  */
+/*         CPC                                                         */
+/*    Call List Logical Partitions of CPC to retrieve uri and          */
+/*         target name and status info for LPARs                       */
+/*    Call Get LPAR Properties to retrieve a set of specific           */
+/*         Image Activation Profile attributes for LPARs               */
+/*         in the specified status (default = operating)               */
+/*         Sample assumes the image activation profile name is         */
+/*         the same as the LPAR name                                   */
```

Comment:
at this point you already know the lpar status ...so you're really just filtering the lpar list returned and using the HWIREST to invoke get image activation profile...would you please re-write this comment

Reply:
Done